test/test_suite.sh

```diff
@@ -29,17 +31,23 @@ else
     WIKIBASE_TEST_IMAGE_NAME="$WIKIBASE_BUNDLE_IMAGE_NAME"
 
     # load additional bundle images
-    docker load -i "../artifacts/$ELASTICSEARCH_IMAGE_NAME.docker.tar.gz"
-    docker load -i "../artifacts/$QUICKSTATEMENTS_IMAGE_NAME.docker.tar.gz"
+    {
+        docker load -i "../artifacts/$ELASTICSEARCH_IMAGE_NAME.docker.tar.gz" >> "$SETUP_LOG" 2>&1
+        docker load -i "../artifacts/$QUICKSTATEMENTS_IMAGE_NAME.docker.tar.gz" >> "$SETUP_LOG" 2>&1
+    } >> "$SETUP_LOG" 2>&1
```

Comment:
I think either the individual or the joined `>> "$SETUP_LOG" 2>&1` can be removed

Reply:
Interestingly I added the grouping as our linting wouldn't allow two lines to have the same output without this. Can't remember the logic as to why, but was fine with it :)